code-challenges.test.js

```diff
@@ -70,4 +160,22 @@ const accountTransactions3 = []
 
 // b) Create the function that makes the test pass.
 
+const arrayAdder = (array) => {
+let sum = 0
+
+for(let i = 0; i < array.length; i++){ 
+sum = array[i] + sum
+array[i] = sum 
+
+}
+  return array
+}
+
+
 // Pseudo code:
+      //Create an array named arrayAdder with a parameter that expects in an array
+      //Making a variable to identify the sum of the upcoming values, we are starting with the number zero
+      //Create a for loop to iterate over array
+      //Reassign the current index plus the sum itself
+      //return the array
```

Comment:
Good pseudo code again here. Make sure your indentation is correct for your code.